draft-ietf-mops-streaming-opcons.md

```diff
@@ -553,10 +553,20 @@ Streaming media latency refers to the "glass-to-glass" time duration, which is t
 
 Streaming media can be usefully categorized according to the application's latency requirements into a few rough categories:
 
-- ultra low-latency    (less than 1 second)
-- low-latency live     (less than 10 seconds)
-- non-low-latency live (10 seconds to a few minutes)
-- on-demand            (hours or more)
+- near realtime latency (less than 150 milliseconds)
+- ultra low-latency     (less than 1 second)
+- low-latency live      (less than 10 seconds)
+- non-low-latency live  (10 seconds to a few minutes)
+- on-demand             (hours or more)
+
+## Near Realtime Latency
+
+Near realtime delivery of media is defined here as having a glass-to-glass (or ear-to-ear) delay target under 150 milliseconds.
+The human ear or eye would mostly not notice or would be on the edge of noticing such latencies.
+These latencies are targeted in various telepresence use cases, such as remote control of drones and autonomous vehicles (e.g. teledriving), remote surgery. For musicians to play together in sync the ideal delay would be below 50 ms.
```

Comment:
I assume there should anyway be some orchestration for this kind of scenario. Like you say, even for an orchestra to play in sync while being in the same room is tricky because of the speed of sound.
There should be a common synchronization source, musicians follow it to play along. And then maybe some post-processing to sync all musicians according to unequal delays.
The exact number is subjective. But I guess the truth is somewhere around 50 ms. As always, the lower latency, the better :)